src/traveler.js

```diff
@@ -1,49 +1,106 @@
 import User from './User'
```

Comment:
I'm curious about what your thinking on extending Traveler from User is? Were there methods that were useful for Traveler to inherent and why could they not be the same class?

Reply:
This is a great question, I am not completely sure yet what are the methods that the `Traveler` class can inherit from the `User` class but the way I visualize it is that, as the app progresses and gets more robust, this could be a good option for when the `traveler` and `agent` start sharing similar behavior. This can become more benefiecial for when the classes start sharing more properties  such as `birthday`, `discounts` , `incentives`, etc...